packages/jaeger-ui/src/components/TracePage/TraceStatistics/PopupSql.test.js

```diff
@@ -13,27 +13,42 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import PopupSql from './PopupSql';
 
-describe('<PopupSQL', () => {
-  let wrapper;
-  let props;
+describe('<PopupSQL />', () => {
+  const popupContent =
+    'select specialtie0_.vet_id as vet_id1_1_0_, specialtie0_.specialty_id as specialt2_1_0_, specialty1_.id as id1_0_1_, specialty1_.name as name2_0_1_ from vet_specialties specialtie0_ inner join specialties specialty1_ on specialtie0_.specialty_id=specialty1_.id where specialtie0_.vet_id=?';
```

Comment:
let's use something simpler that fits on one line / width. There's no reason for such complex query in the test.
```suggestion
    'SELECT * FROM users WHERE user_id=?';
```